docker-compose.initdb.yml

```diff
@@ -0,0 +1,8 @@
+version: '3.4'
+
+services:
```

Comment:
Can we just move this to the one `docker-compose` so developers don't have to run two commands? (additionally you'd probably want to change the script to include waiting for the DB to be up).

Reply:
inital version was just one docker-compose but then you have to wait everytime to import the data into database which is not necessary everytime.

Vast manority of the time you are going to do changes on client / server, so when you do `docker-compose up` you don't need to import the database again